scripts/wget/1.19.2/script.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+MASON_NAME=wget
+MASON_VERSION=1.19.2
+MASON_LIB_FILE=bin/wget
+
+. ${MASON_DIR}/mason.sh
+
+function mason_load_source {
+    mason_download \
+        http://ftp.gnu.org/gnu/wget/wget-1.19.2.tar.gz \
```

Comment:
Best practice is to do `http://ftp.gnu.org/gnu/wget/${MASON_NAME}-${MASON_VERSION}.tar.gz` such that creating new package versions, based on `cp -r <this package> <new package`, is easier.

Reply:
👍 Fixed